hax/hax/ha/events.py

```diff
@@ -75,9 +77,21 @@ def __init__(self,
         if topic is None:
             raise RuntimeError('Failed to subscribe to events')
 
-        message_bus = MessageBus()
+        util: ConsulUtil = ConsulUtil()
+        configpath = util.get_configpath()
+        Conf.load('cortx_conf', configpath)
+
+        message_server_endpoints = Conf.get('cortx_conf',
+                                            'cortx>external>kafka>endpoints')
+
+        """
+        This creates a MessageBus internal global in-memory object which is
+        indirectly part of the process address space. Thus
+        `MessageBus.init()` must be invoked within process's address space.
+        """
+        MessageBus.init(message_server_endpoints)
```

Comment:
@SwapnilGaonkar7 , I think this should be moved to hax.py::main(), please introduce a new function, ha_message_bus_init() and move the code in it.